File: src/main/kotlin/com/github/frcsty/frozenjoin/action/util/UtilServer.java
package com.github.frcsty.frozenjoin.action.util;

import com.google.common.io.ByteArrayDataOutput;
import com.google.common.io.ByteStreams;
import org.bukkit.entity.Player;
import org.bukkit.plugin.Plugin;

import java.util.Arrays;

public final class UtilServer {

    public static void writeToBungee(final Player player, final Plugin plugin, final String... args) {
        ByteArrayDataOutput output = ByteStreams.newDataOutput();
        Arrays.stream(args).forEach(output::writeUTF);

        player.sendPluginMessage(plugin, "BungeeCord", output.toByteArray());
    }
}

File: src/main/kotlin/com/github/frcsty/frozenjoin/action/util/UtilServer.kt
package com.github.frcsty.frozenjoin.action.util

import com.google.common.io.ByteStreams
import org.bukkit.entity.Player
import org.bukkit.plugin.Plugin
import java.util.*

object UtilServer {
    fun writeToBungee(player: Player, plugin: Plugin, vararg args: String) {
        val output = ByteStreams.newDataOutput()
        Arrays.stream(args).forEach { s: String -> output.writeUTF(s) }
        player.sendPluginMessage(plugin, "BungeeCord", output.toByteArray())
    }
}